src/containers/products/ProductScreen.js

```diff
@@ -14,13 +15,23 @@ export const ProductScreen = () => {
 	}, [dispatch]);
 
 	return (
-		<div>
-			<h1>Products</h1>
-			<ProductContainerStyle primary>
-				{products.map((item, index) => (
-					<ProductItem key={index} item={item} />
-				))}
-			</ProductContainerStyle>
-		</div>
+		<>
+			{loading ? (
+				<h1>Loading...</h1>
+			) : error ? (
+				<div>{error}</div>
+			) : (

```

Comment:
I think, the best code style for this case would be like:

```js
if (loading) {
  return ( 
   <h1>Loading...</h1>
  )
}
```
If you think the same, then put it in a separate function. For example:  **handleRenderProducts**